image_explorer/templates/html/image_explorer.html

```diff
@@ -36,6 +37,15 @@ <h2>{{title}}</h2>
                 </div>
             </div>
         {% endfor %}
+        </div>
+        <div class="tutorial-wrapper">
+            <div class="tutorial-container">
+                <h3>Instructions</h3>
+                <p>Click each step in the loop to learn more about the problem solving approach. When you are finished, click the forward arrow to continue.</p>
+                <div class="tutorial-video">&nbsp;</div>
+                <a id="hide-tutorial">Start Excercise</a>
```

Comment:
This should be "Exercise". I think there's an extra 'c' in there.
Sorry I didn't notice this before.